src/tokens/tokens.controller.ts

```diff
@@ -1,31 +1,21 @@
 import { Network } from '@badger-dao/sdk';
-import { Controller, Get, QueryParams } from '@tsed/common';
+import { Controller, Get, QueryParams, UseCache } from '@tsed/common';
 import { ContentType, Description, Returns, Summary } from '@tsed/schema';
 import { Chain } from '../chains/config/chain.config';
 import { TokenConfigModel } from './interfaces/token-config-model.interface';
 
 @Controller('/tokens')
 export class TokensController {
   @Get()
+  @UseCache()
   @ContentType('json')
   @Summary('Get a summary of tokens related to the Badger Protocol')
   @Returns(200, TokenConfigModel)
   @Description('Return a map of checksum contract address to token information.')
   async listSetts(@QueryParams('chain') chain?: Network): Promise<TokenConfigModel> {
-    return Object.fromEntries(
-      Object.entries(Chain.getChain(chain).tokens).map((entry) => {
-        const [key, value] = entry;
-        const { name, symbol, decimals, address } = value;
-        return [
-          key,
-          {
-            name,
-            symbol,
-            decimals,
-            address,
-          },
-        ];
-      }),
-    );
+    const requestChain = Chain.getChain(chain);
+    const chainTokens = Object.keys(requestChain.tokens);
+    const sdk = await requestChain.getSdk();
+    return sdk.tokens.loadTokens(chainTokens);
```

Comment:
can we utilize the db on the outgoing call and the sdk on the async updates of tokns